zend_abstract_interface/jit_utils/jit_blacklist.c

```diff
@@ -185,6 +178,60 @@ int zai_get_zend_func_rid(zend_op_array *op_array) {
     return zend_func_info_rid;
 }
 
+#if defined(__linux__) && (defined(__x86_64__) || defined(__aarch64__)) && PHP_VERSION_ID < 80400
```

Comment:
We do not ship 32bit binaries and AFAIK we do not (officially) support it, but in case some compiles this on a armv7 (bit older raspi) this wont compile as there is not `is_mapped()` function.

Reply:
To add to that: I have no idea if one could currently compile on 32 bit hardware anyway 😉